browse/config.py

```diff
@@ -224,3 +224,6 @@
 
 # Used in linking to /show-email
 SHOW_EMAIL_SECRET = os.environ.get('SHOW_EMAIL_SECRET', 'foo')
+
+# Used in linking to /ct
+CLICKTHROUGH_SECRET = os.environ.get('CLICKTHROUGH_SECRET', 'bar')
```

Comment:
There is no need for a separate clickthrough secrete key.
 app.config.get('SECRET_KEY') can be used.